src/components/10-atoms/input-file/index.scss

```diff
@@ -1,3 +1,7 @@
+axa-input-file {
```

Comment:
This selector could be invalid if someone uses the new component versioning like `axa-input-file-pod_xy`. Better set display to the internal elements or create a wrapper div for that.

Reply:
As discussed before, this shouldn't be a problem.